### ISSUE TITLE
Implement content and user feedback changes

### ISSUE THREAD

**Issue by austinhernandez** [18F/clinical-trials#134]
As we wrap up this engagement and focus on the deliverables, the design team will start to hone in on features to test and "finalize" for prototyping by @maya 

a brief to do list of 

- [x] *study record detail page* Mel takes a crack at gray box at top
- [x] Mel shares with Austin for feedback
- [x] *search results* Austin refine filters and "modify search" options to match homepage
- [x] Austin puts these into Invision by 10:30a for Mel to test tomorrow
- [ ] refine *search results* and *study record detail page* based on what we learn tomorrow (@MelissaBraxton - what did we learn?)
- [ ] Finalize content with @jamiealbrecht 
- [ ] Share with Maya for feasibility critique
- [ ] Refine based on Maya's feedback
- [ ] Send to Maya to prototype in time to test during our final round (@maya are we even gonna have your time to work on this since we are extending the engagement?)

**Comment by MelissaBraxton** [comment]
We have 2 interviews that had to be rescheduled to next week, aiming to be able to summarize what we've learned by the afternoon of 7/10.

**Comment by maya** [comment]
>@maya are we even gonna have your time to work on this since we are extending the engagement?)

I defer to @acolter about my utilization for the rest of the project, but if the team needs me to prototype this then I think I should spend time on that.

**Comment by MelissaBraxton** [comment]
Yes, for @acolter  - We are counting on having @maya help prototype the second half of next week, but very minimal (if any) prototyping after that. cc @austinhernandez
